docs/en/observability/gcp-topic.asciidoc

```diff
@@ -0,0 +1,19 @@
+There are three available filesets:
+`audit`, `vpcflow`, `firewall`. This tutorial covers the `audit` fileset.
+
+. Go to the *Logs Router* page to configure {gcp} to export logs to a Pub/Sub
+topic. Use the search bar to find the page:
++
+image::monitor-gcp-navigate-logs-router.png[Navigate to Logs Router page]
++
+To set up the logs routing sink, click  *Create sink*.
+Set *sink name* as `monitor-gcp-audit-sink`. Select the *Cloud Pub/Sub topic* as the
+*sink service* and *Create new Cloud Pub/Sub topic* named `monitor-gcp-audit`:
++
+image::monitor-gcp-create-pubsub-topic.png[Create Pub/Sub topic]
++
+Finally, under *Choose logs to include in sink*, add
+`logName:"cloudaudit.googleapis.com"` (it includes all audit logs).
+Click *create sink*.  It will look something like the following:
++
+image::monitor-gcp-create-sink.png[Create logs routing sink]
```

Comment:
I wonder if this document needs two screen captures that are fairly similar? Screen captures of third-party tools can be hard to keep up-to-date over time, so it's best to limit them, if possible.

I'd show the finished screen rather than in-progress.